### PR TITLE
Add sensor_reading and sensor_value helpers

### PR DESCRIPTION
**Pull request by bmerry**
These simplify the process of getting the current reading or value of a sensor.

### CODE REVIEW

File: src/aiokatcp/client.py [hunk 1]
```diff
@@ -615,6 +618,88 @@ async def request(self, name: str, *args: Any) -> Tuple[List[bytes], List[core.M
...
+    @overload
+    async def sensor_value(self, sensor_name: str, sensor_type: None = None) -> Any:
+        ...
+
+    @overload
+    async def sensor_value(self, sensor_name: str, sensor_type: Type[_T]) -> _T:
+        ...
+
+    async def sensor_value(self, sensor_name: str, sensor_type: Optional[type] = None) -> Any:
+        """Request the reading of a single sensor from the server.
```
Comment (james-smith-za) [inline src/aiokatcp/client.py:688]:
```suggestion
        """Request the value of a single sensor from the server.
```
?

**Review by james-smith-za — APPROVED** [review]
Happy. One nit-pick, feel free to merge after addressing.

File: src/aiokatcp/client.py [hunk 2]
```diff
@@ -224,7 +224,7 @@ async def handle_message(self, conn: connection.Connection, msg: core.Message) -
                 elif msg.mtype == core.Message.Type.INFORM:
                     req.informs.append(msg)
                 else:
-                    self.logger.warning("Unknown message type %s", msg.mtype)  # pragma: no cover
```
Comment (james-smith-za) [inline src/aiokatcp/client.py:224]: I mean this one looks as though we can probably explicitly test for it but it would be a bit contrived.

**Review by james-smith-za — APPROVED** [review]

File: src/aiokatcp/client.py (hunk 2 above)
Comment (bmerry) [reply src/aiokatcp/client.py:224]: Sure it's possible, but also outside the scope of this PR.

**Pull request merged**